lib/ohai/plugins/linux/platform.rb

```diff
@@ -53,7 +53,7 @@ def get_redhatish_platform(contents)
   # @returns [String] the version string
   #
   def get_redhatish_version(contents)
-    contents[/Rawhide/i] ? contents[/((\d+) \(Rawhide\))/i, 1].downcase : contents[/(release)? ([\d\.]+)/, 2]
```

Comment:
Can you add a spec here so we make sure we don't regress this later on. Also it's really useful to add a https://rubular.com/ permalink to the comments that shows the various things the regex is trying to match. That way anyone trying to adjust this regex later on can easily test the matches. We try to add those any time we touch a regex now.